<launch>
  <node pkg="ccms_pro" type="ProcessData2.py" name="block_detect_subscribe" output="screen"/>
  <node pkg="ccms_pro" type="Block_Detect.py" name="block_detect" output="screen"/>
  <node pkg="ccms_pro" type="Module_Detect.py" name="module_detect" output="screen"/>
  <node pkg="ccms_pro" type="Battery_Detect.py" name="battery_detect" output="screen"/>
</launch>

